virtcontainers: Add to bridges unit test

cmaf opened the pull request:
Add function that creates new bridges to increase unit test coverage
for virtcontainers/types/bridges.

Fixes #2827

Signed-off-by: Chelsea Mafrica <chelsea.e.mafrica@intel.com>

cmaf commented:
I am still adding to this for more coverage for this unit test but a review is appreciated.

cmaf commented:
/test-ubuntu

cmaf commented:
/test-ubuntu

jcvenegas commented on virtcontainers/types/bridges_test.go:
```diff
@@ -43,6 +43,40 @@ func testAddRemoveDevice(t *testing.T, b *Bridge) {
+	var ccw Type = "ccw"
+
+	var pcibridge = NewBridge(PCI, "", make(map[uint32]string), 0)
+	assert.Equal(pcibridge.Type, pci, "Type should be PCI")
```
You could also assert the values of map and addr

cmaf commented on virtcontainers/types/bridges_test.go:
```diff
@@ -43,6 +43,40 @@ func testAddRemoveDevice(t *testing.T, b *Bridge) {
+	assert := assert.New(t)
+
+	var ccwbridge = NewBridge(CCW, "", make(map[uint32]string), 0)
+	_, err := ccwbridge.AddressFormatCCW("0")
```
Applied these changes

cmaf commented:
@jcvenegas I updated the test. I added a test for the default bridge and I think I hit 100% coverage. 

Should I add a test for every parameter? I left `ID` as an empty string but I could test it, and I only test `maxCapacity` for the default bridge.

cmaf commented:
/test-ubuntu

cmaf commented:
/test-ubuntu

jodh-intel commented:
@cmaf - could you port this to the https://github.com/kata-containers/kata-containers/tree/2.0-dev branch too please.

cmaf commented:
Ported to 2.0 with [423](https://github.com/kata-containers/kata-containers/pull/423)
